lib/tasks/ingestor.rake

```diff
@@ -124,38 +124,50 @@ namespace :fromthepage do
     # first process all sub-directories
     ls = Dir.glob(File.join(temp_dir, "*"))
     ls.each do |path|
-      print "ingest_tree handling #{path})\n"
+      print "ingest_tree considering #{path})\n"
       if Dir.exist? path
         print "Found directory #{path}\n"
         ingest_tree(document_upload, path) #recurse
       end
     end    
     # now process this directory if it contains image files
-    image_files = Dir.glob(File.join(temp_dir, "*{"+IMAGE_FILE_EXTENSIONS.join(',')+"}"))
+    image_files = Dir.glob(File.join(temp_dir, "*.{"+IMAGE_FILE_EXTENSIONS.join(',')+"}"))
```

Comment:
This was the source of the empty file.  Because we were looking for "image files" in a directory before processing it, and because we were defining "image files" as files that end with `jpg`, and because the zip file UT uploaded happened to have a name ending with `jpg`, we got a false positive on the root directory for the upload.

I think adding the period should fix this.